components/notebook-controller/config/manager/manager.yaml

```diff
@@ -32,7 +32,27 @@ spec:
               configMapKeyRef:
                 name: config
                 key: ISTIO_GATEWAY
-        imagePullPolicy: Always
+          - name: CLUSTER_DOMAIN
+            valueFrom:
+              configMapKeyRef:
```

Comment:
Configmap ref is added to the deployment. We should make sure, it matches with the configmap created by[ odh-dashboard](https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts#L73).